storage/postgres/migrations/20250224075309_add_additional_columns_to_plugin_policy.sql

```diff
@@ -8,10 +14,13 @@ ALTER TABLE plugin_policies
 ADD COLUMN derive_path TEXT NOT NULL;
 ALTER TABLE plugin_policies
 ADD COLUMN active BOOLEAN DEFAULT TRUE;
+ALTER TABLE plugin_policies
```

Comment:
a potential conflict with migrations from https://github.com/vultisig/vultiserver-plugin/pull/90

depending on which PR enters in `develop-pre-dkls` first